function-aws-api-proxy-test/build.gradle.kts

```diff
@@ -5,7 +5,6 @@ plugins {
 dependencies {
     api(mn.micronaut.http.server)
     api(projects.micronautFunctionAwsApiProxy)
-    implementation(platform(mnServlet.boms.jetty))
```

Comment:
In this PR, I changed the build to no longer depend on the version of jetty from servlet. The `micronaut-function-aws-api-proxy-test` code assumes Jetty is still 11. We should update the code to Jetty 12. But I suggest doing it in a different PR.